src/main/java/codeu/model/data/User.java

```diff
@@ -24,6 +24,7 @@
   private String aboutMe;
   private final String passwordHash;
   private final Instant creation;
+  private final Boolean privileges;
```

Comment:
How about isAdmin here and elsewhere? That seems to be more consistent with the existing terminology.

Also it seems like you might want to use boolean instead of Boolean so that null isn't allowed.